Bottom: 07bc5862ed6c2545c842cffec9d51bf25b79fb40
Top:    1b10902e7fa295ebd049a0b67cce04b6350ad096
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-09-06 23:09:35 -0400

build command


---

diff --git a/cop b/cop
new file mode 100755
index 0000000..6198c36
--- /dev/null
+++ b/cop
@@ -0,0 +1 @@
+make uImage
